myapp.rb

```diff
@@ -131,8 +154,7 @@ def current_user
     if Message.create(
       author_id: session[:user_id],
       text: params[:text],
-      pub_date: Time.now,
```

Comment:
Does ActiveRecords automatically generate a timestamp for `created_at`?

Reply:
In our migration "t.timestamps" automatically creates the columns "created_at" and "updated_at" which automatically gets set accordingly